core/isolate.rs

```diff
@@ -1140,6 +1140,7 @@ pub mod tests {
   }
 
   #[test]
+  #[ignore]
```

Comment:
I’d like to land this for the perf improvement but I don’t think these tests should be disabled - if they aren’t working I think this change is introducing a bug.

Reply:
Yeah, I'll split it into 2 separate PRs: one for shared queue and one for dispatch_minimal